addons/website_sale_stock/tests/common.py

```diff
@@ -0,0 +1,32 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests import TransactionCase
+
+
+class TestWebsiteSaleStockCommon(TransactionCase):
+
+    @classmethod
+    def add_product_qty_to_wh(cls, product_id, qty, loc_id):
+        cls.env['stock.quant'].with_context(inventory_mode=True).create({
+            'product_id': product_id,
+            'inventory_quantity': qty,
+            'location_id': loc_id,
+        }).action_apply_inventory()
+
+    @classmethod
+    def create_storable_product_product(cls, **kwargs):
+        return cls.env['product.product'].create({
+            'name': 'Test',
+            'is_storable': True,
+            'allow_out_of_stock_order': False,
+            'website_published': True,
+            **kwargs,
+        })
+
+    @classmethod
+    def create_warehouse(cls, **kwargs):
+        return cls.env['stock.warehouse'].create({
+            'name': 'Warehouse 2',
```

Comment:
```suggestion
            'name': 'Test Warehouse',
```

I don't believe having a numbering makes sense in a common, even less when starting on 2.
I'm aware it might break some existing tests you're trying to convert to this new common, but new common must be created with the generic purpose in mind, not to ease the conversion of existing test classes.  I wouldn't mind if you created the common without adapting the existing tests to it, but a new common must be as clean and easy to use as possible.